websites/Y/YouTube/dist/metadata.json

```diff
@@ -120,7 +120,7 @@
       },
       "title": "Show Channel's Picture",
       "icon": "fas fa-user",
-      "value": false
+      "value": true
```

Comment:
Why did you make this change?

Reply:
oops, forgot it while i was testing it so i dont have to turn it on every time